resolve/LinSolverIterativeFGMRES.hpp

```diff
@@ -37,11 +37,13 @@ namespace ReSolve
     index_type getMaxit();
     index_type getRestart();
     index_type getConvCond();
+    bool getFlexible();
```

Comment:
```suggestion
    bool getFlexible() const;
```

We should probably make these pure getters const to reflect their behavior.

Reply:
Tracked in #89